src/os2datascanner/projects/admin/tests/test_miniscanner.py

```diff
@@ -0,0 +1,135 @@
+from os2datascanner.projects.admin.adminapp.views.miniscanner_views import *
+from random import choice
+
+class DummyFile():
+    def __init__(self, name, fileContents) -> None:
+        self.contents = fileContents
+        self.name = name
+
+    def read(self):
+        return self.contents.encode()
+
+class FilesObj():
+    def __init__(self, objs) -> None:
+        self.files = objs
+    def get(self, obj):
+        return self.files[obj]
+    
+class Post():
+    def __init__(self, objs) -> None:
+        self.items = objs
+    def get(self, obj):
+        return self.items[obj]
+    
+class Meta():
+    def __init__(self, objs) -> None:
+        self.items = objs
+    def get(self, obj):
+        return self.items[obj]
+
+class Request():
+    def __init__(self, fileObjsDict, postObjsDict, metaObjsDict) -> None:
+        self.FILES = FilesObj(fileObjsDict)
+        self.POST = Post(postObjsDict)
+        self.META = Meta(metaObjsDict)
+
+def generateDummyContent(n=10):
+    CHARACTERS = "abcdefghijklmnopqrstuvwxyzæøå"
+    return '\n'.join([''.join([choice(CHARACTERS) for _ in range(25)]) for _ in range(n)]) # Generates a block of random characters
```

Comment:
`choice(CHARACTERS) for _ in range(25)` is rather a complicated way of saying `choices(CHARACTERS, k=25)`. :-)

Reply:
I actually didn't know about the k argument, thanks for letting me know :D. I'll fix that very soon !

Reply:
It works very well, and is much cleaner. Fixed.